plugins/mcreator-localization/lang/texts.properties

```diff
@@ -567,6 +567,11 @@ workspace.category.remote_workspace=Remote  workspace
 workspace.elements.add.tooltip=Add new mod element
 workspace.elements.edit.tooltip=Edit selected mod element
 workspace.elements.duplicate.tooltip=Duplicate/copy selected mod element
+workspace.elements.details.name=Element name
+workspace.elements.details.id=Element ID
```

Comment:
Unless this saves some space, rename this to _Element registry name_.

Reply:
Done